### PR TITLE
[POAE7-2401]modify presto documenttation

### PR DESCRIPTION
**Pull request by PenghuiJiao**
please take review @winningsix  @xieqi @Ma-Jian1 
### What changes were proposed in this pull request?

modify docs/develop-guide.rst

### Why are the changes needed?

Documentation update

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

No

### Which label does this PR belong to?

cider

### CODE REVIEW

**Comment by winningsix** [comment]
It's not a hands-on doc for entry-level users. Imaging you're an user of Presto engine and come to our lib, how should I start with this?
On top of my mind, a few questions to answer:
- Where to download Presto package? or self build?
- Any impacts of normal Presto usage? e.g., Presto-cli or some others
- Any utility script I can directly use to start Presto? Is https://github.com/intel/BDTK/pull/58/files#diff-8069a52cbef040e62e22ba5c1066523c09f2caec9d8e1af441eb9bcddb6eeee6R350 the only thing to do?
- Besides that, we should consider some validation steps in doc to ensure previous steps are correct? e.g., when started Presto, how can we tell our lib is loaded correctly?

File: docs/develop-guide.rst [hunk 1]
```diff
@@ -336,38 +336,24 @@ Start the CLI to connect to the server and run SQL queries:
 7 How to run simple examples with Prestodb in distributed environment
 ---------------------------------------------------------------------
 
-(1) Create a folder to install cider files, for example ``cider``
-
-(2) | Copy the lib folder under the cider docker build environment to
-      every node, for example, copy
-      ``/usr/local/lib/`` folder to
-      ``/path/to/cider`` on every Prestodb node
+(1) Unzip the Prestodb package and enter the unzip package
```
Comment (Ma-Jian1) [inline docs/develop-guide.rst:339]: where to get the package?

File: docs/develop-guide.rst [hunk 2]
```diff
@@ -336,38 +336,109 @@ Start the CLI to connect to the server and run SQL queries:
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 ::
 
-   cp -a /usr/local/lib/ /path/to/cider
+   git clone --recursive https://github.com/Intel-bigdata/presto.git
+   pushd presto
+   git checkout -b BDTK origin/BDTK
+   pushd presto-native-execution
+   git clone --recursive https://github.com/intel/BDTK.git
```
Comment (Ma-Jian1) [inline docs/develop-guide.rst:348]: `git clone --recursive https://github.com/intel/BDTK.git velox-plugin` then you don't need change include path in presto

Comment (PenghuiJiao) [reply docs/develop-guide.rst:348]: Now our project name is BDTK, why use the folder name velox-plugin?

**Review by Ma-Jian1 — APPROVED** [review]

File: docs/develop-guide.rst [hunk 3]
```diff
@@ -336,38 +336,109 @@ Start the CLI to connect to the server and run SQL queries:
...
+   pushd BDTK  
+   make release
+   if [ $? -ne 0 ]; then
```
Comment (xieqi) [inline docs/develop-guide.rst:351]: This is a user guide, not the script, don't need such check

Comment (PenghuiJiao) [reply docs/develop-guide.rst]: fix

File: docs/develop-guide.rst [hunk 4]
```diff
@@ -336,38 +336,101 @@ Start the CLI to connect to the server and run SQL queries:
 7 How to run simple examples with Prestodb in distributed environment
 ---------------------------------------------------------------------
 
-(1) Create a folder to install cider files, for example ``cider``
-
-(2) | Copy the lib folder under the cider docker build environment to
-      every node, for example, copy
-      ``/usr/local/lib/`` folder to
-      ``/path/to/cider`` on every Prestodb node
+7.1 build presto in container
```
Comment (xieqi) [inline docs/develop-guide.rst:339]: build presto native execution?

File: docs/develop-guide.rst [hunk 5]
```diff
@@ -336,38 +336,101 @@ Start the CLI to connect to the server and run SQL queries:
...
+   sed -i 's/\"planTransformer\/PlanTransformer\.h\"/\"..\/planTransformer\/PlanTransformer\.h\"/' ./BDTK/cider-velox/src/ciderTransformer/CiderPlanTransformerFactory.h
+   sed -i 's/\"velox-plugin\/cider-velox\/src\/CiderVeloxPluginCtx.h\"/\"BDTK\/cider-velox\/src\/CiderVeloxPluginCtx.h\"/' ./presto_cpp/main/TaskResource.cpp
+   sed -i 's/\"velox-plugin\/cider-velox\/src\/CiderVeloxPluginCtx.h\"/\"BDTK\/cider-velox\/src\/CiderVeloxPluginCtx.h\"/' ./presto_cpp/main/PrestoServer.cpp
+   make PRESTO_ENABLE_PARQUET=ON -j ${CPU_COUNT:-`nproc`} release
+   mkdir -p ./_build/release/presto_cpp/function
+   cp ./BDTK/build-Release/cider/function/RuntimeFunctions.bc ./_build/release/presto_cpp/function/
+   popd
+   popd
+
+7.2 build presto package
```
Comment (xieqi) [inline docs/develop-guide.rst:371]: presto native execution?

File: docs/develop-guide.rst [hunk 6]
```diff
@@ -336,38 +336,101 @@ Start the CLI to connect to the server and run SQL queries:
...
-(8) You may also need include the libjvm.so in your LD_LIBRARY_PATH if
-    it is not
+7.5 run presto_server
+~~~~~~~~~~~~~~~~~~~~~
 
 ::
-
-   export LD_LIBRARY_PATH= $JAVA_HOME/jre/lib/amd64/server/:$LD_LIBRARY_PATH
+   
+   ./bin/presto_server
```
Comment (xieqi) [inline docs/develop-guide.rst:433]: will work? Need run with parameter point to etc folder?

**Review by xieqi — DISMISSED** [review]

**Comment by winningsix** [comment]
@PenghuiJiao Please address the conflicts.

File: docs/develop-guide.rst [hunk 7]
```diff
@@ -336,38 +336,101 @@ Start the CLI to connect to the server and run SQL queries:
...
+   git clone --recursive https://github.com/Intel-bigdata/presto.git
+   pushd presto
+   git checkout -b BDTK origin/BDTK
+   pushd presto-native-execution
+   git clone --recursive https://github.com/intel/BDTK.git
+   pushd BDTK  
+   make release
+   popd
+   mkdir -p ./presto_cpp/main/lib
+   cp ./BDTK/build-Release/cider-velox/src/libvelox_plugin.a ./presto_cpp/main/lib
```
Comment (winningsix) [inline docs/develop-guide.rst:353]: We should provide a script to do deployment instead file-by-file copying so many so files. Your thoughts?

**Review by Ma-Jian1 — APPROVED** [review]

File: docs/develop-guide.rst (hunk 7 above)
Comment (PenghuiJiao) [reply docs/develop-guide.rst]: fix, add build-presto-package.sh

File: docs/develop-guide.rst (hunk 6 above)
Comment (PenghuiJiao) [reply docs/develop-guide.rst]: fix

File: docs/develop-guide.rst (hunk 5 above)
Comment (PenghuiJiao) [reply docs/develop-guide.rst]: removed

File: docs/develop-guide.rst (hunk 4 above)
Comment (PenghuiJiao) [reply docs/develop-guide.rst]: removed

**Review by winningsix — APPROVED** [review]

File: ci/scripts/build-presto-package.sh [hunk 8]
```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```
Comment (winningsix) [inline ci/scripts/build-presto-package.sh:1]: Sorry. Forgot to mention: copyright and license header needed here.

Comment (PenghuiJiao) [reply ci/scripts/build-presto-package.sh:1]: fix

**Review by Ma-Jian1 — APPROVED** [review]

**Review by winningsix — APPROVED** [review]

**Pull request merged**